tilewe/__init__.py

```diff
@@ -249,3 +249,8 @@ def is_legal(self, move: Move, for_player: Color=None) -> bool:
 ]
 
 from ctilewe import *  # noqa: E402, F401, F403
+
+PIECE_TILES:    list[int] = [n_piece_tiles(piece)    for piece in range(NO_PIECE)]
+PIECE_CORNERS:  list[int] = [n_piece_corners(piece)  for piece in range(NO_PIECE)]
+PIECE_CONTACTS: list[int] = [n_piece_contacts(piece) for piece in range(NO_PIECE)]
```

Comment:
Are the cached lists actually faster? The C function should be pretty fast. 

Also I feel like it would be better to use `len(PIECES)` instead of the null piece's value, which is set to piece count for convenience and not because that necessarily should be its definition.

Reply:
TBD